websurfx/config.lua

```diff
@@ -14,6 +14,8 @@ rate_limiter = {
 	number_of_requests = 20, -- The number of request that are allowed within a provided time limit.
 	time_limit = 3, -- The time limit in which the quantity of requests that should be accepted.
 }
+-- Set whether the server will use an adaptive/dynamic TCP window size
+adaptive_window = true
```

Comment:
```suggestion
-- Set whether the server will use an adaptive/dynamic HTTPS window size
https_adaptive_window_size = true
```

I would suggest renaming the config option to `https_adaptive_window_size` as I think it will better represent what the config option actually does and also it may make it easier for the user to understand as well. Moreover, this option actually effects the `https` window size rather than the `tcp` window size. So I would suggest updating the documentation for the same as well. :slightly_smiling_face:

Reply:
Are you sure about it being an https window size. I can't find any information about that when searching, the only thing that comes up is the TCP window size.

Reply:
> Are you sure about it being an https window size. I can't find any information about that when searching, the only thing that comes up is the TCP window size.

Yes, because, you can already see it from the settings option `ClientBuilder` struct like it changes the window size for `https/http2` rather than the `tcp` window size. :slightly_smiling_face: